refextract/references/kbs.py

```diff
@@ -42,6 +43,23 @@
 from ..documents.text import re_group_captured_multiple_space
 
 
+@contextlib.contextmanager
+def file_resolving(fpath, reader=None, **kwargs):
+    if isinstance(fpath, six.string_types):
+        print('Load kb path %s' % fpath)
```

Comment:
@jacquerie https://github.com/zzacharo/refextract/blob/e1dc83392e37698f206d27144eb57db9a04dd65c/refextract/references/kbs.py#L487 this print how should be handled?